sky/templates/azure-ray.yml.j2

```diff
@@ -101,23 +107,31 @@ setup_commands:
     sudo pkill -9 apt-get;
     sudo pkill -9 dpkg;
     sudo dpkg --configure -a;
-    sudo sed -i 's/#user_allow_other/user_allow_other/g' /etc/fuse.conf;
+    [ -f /etc/fuse.conf ] && sudo sed -i 's/#user_allow_other/user_allow_other/g' /etc/fuse.conf || (sudo sh -c 'echo "user_allow_other" > /etc/fuse.conf'); # This is needed for `-o allow_other` option for `goofys`;
```

Comment:
This command is most likely not neccessary given that `Azure Blob` isn't supported for Sky Storage atm.

Reply:
It's kept here for consistency with other public cloud templates.